src/cljs/main/broadfcui/main.cljs

```diff
@@ -239,7 +239,7 @@
                                                      "FireCloud service is temporarily unavailable.")]
              (notifications/render-alert {:title title
                                           :message msg
-                                          :link "http://status.firecloud.org/"
+                                          :link (config/status-url)
```

Comment:
@aweng98 are you referring to this change specifically? Please let me know, I'm happy to revert if the alternative might cause flakiness.